### ISSUE TITLE
Package path ./runtime.js is not exported from package

### ISSUE THREAD

**Issue by apostolos** [pmmmwh/react-refresh-webpack-plugin#565]
`react-refresh` started using the exports field in package.json
https://github.com/facebook/react/commit/2f26eb85d657a08c21edbac1e00f9626d68f84ae

Since then, the following error is thrown:
```
[0] ERROR in ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js 4:23-58
[0] Module not found: Error: Package path ./runtime.js is not exported from package ./node_modules/react-refresh
(see exports field in ./node_modules/react-refresh/package.json)
```

**Comment by pmmmwh** [comment]
> `react-refresh` started using the exports field in package.json
> 
> https://github.com/facebook/react/commit/2f26eb85d657a08c21edbac1e00f9626d68f84ae
> 
> 
> 
> Since then, the following error is thrown:
> 
> ```
> 
> [0] ERROR in ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js 4:23-58
> 
> [0] Module not found: Error: Package path ./runtime.js is not exported from package ./node_modules/react-refresh
> 
> (see exports field in ./node_modules/react-refresh/package.json)
> 
> ```
> 
> 

I believe this is not published yet?

**Comment by apostolos** [comment]
It is on the `@experimental` channel.

**Comment by apostolos** [comment]
@pmmmwh React 18 is now published.
https://reactjs.org/blog/2022/03/29/react-v18.html

**Comment by pmmmwh** [comment]
Released in 0.5.5
